components/ai_chat/resources/page/components/page_context_toggle/index.tsx

```diff
@@ -22,8 +22,7 @@ function PageContextToggle() {
     context.updateShouldSendPageContents(checked)
   }
 
-  const toggleTooltipVisibility = (e: any) => {
-    e.preventDefault()
```

Comment:
@nullhook could you confirm this won't break anything?

Reply:
this works!